src/runtime_src/core/pcie/linux/scan.cpp

```diff
@@ -318,7 +318,7 @@ put(const std::string& name,
   if (!err.empty())
     return;
   fs << input;
-  fs.flush();
+  fs.close();
   if (!fs.good()) {
```

Comment:
It doesn't make any sense to me to check fs.good() after closing the stream.
Please clarify.

Reply:
My mistake; you are right. I didn't see that fs.good() became 0 after closing the stream.

Reply:
Actually Ryan, checking good() after close() is correct.  Just looked strange at first glance, so maybe a comment would be appropriate, something like:
`fs.close();  // flush and close, if either fails then stream failbit is set.`

Reply:
Did so, thank you!